packages/shared/src/api_types.ts

```diff
@@ -4,10 +4,16 @@ import {
   ITimeControlConfig,
 } from "./time_control/time_control.types";
 
+export interface UserRanking {
+  rating: number;
+  rd: number;
+  vol: number;
+}
+
 export interface User {
   username?: string;
   id: string;
-  rating?: number;
+  ranking?: UserRanking;
```

Comment:
Should we have this per variant?  Like

```
rankings: { [variant: string]: UserRanking }
```

Example:

```
{ "quantum": { rating: 1500, rd: 350, vol: rd: 0.06} }
```

Reply:
Yes!